mmdet/datasets/pipelines/transforms.py

```diff
@@ -295,7 +295,7 @@ def _pad_masks(self, results):
         pad_shape = results['pad_shape'][:2]
         for key in results.get('mask_fields', []):
             results[key] = results[key].pad(
-                pad_shape[:2], pad_val=self.pad_val)
+                pad_shape, pad_val=self.pad_val)
```

Comment:
Please use yapf to format the code following the [contributing guide](https://github.com/open-mmlab/mmdetection/blob/master/.github/CONTRIBUTING.md).